src/main/external-resources/renderers/Google-ChromecastUltra.conf

```diff
@@ -41,12 +41,5 @@ Supported = f:m4a            n:2      a:aac-lc|he-aac|mp3   m:audio/mp4
 Supported = f:wav            n:2      a:lpcm                m:audio/x-wav
 Supported = f:weba           n:2      a:opus|vorbis         m:audio/webm
 
-# Supported image formats:
```

Comment:
My preference would be not to do individual adjustments unless absolutely necessary. Chromecast doesn't seem to support DLNA at all, but use the [DIAL protocol](http://www.dial-multiscreen.org/dial-protocol-specification) instead. I doubt Chromecast will ever work "properly" with UMS as long as the DIAL protocol isn't implemented. It should be handled separately from DLNA.